common/test/acceptance/tests/studio/test_studio_html_editor.py

```diff
@@ -100,3 +113,186 @@ def test_link_plugin_sets_url_correctly(self):
+            And I save the page
+                Then the page text contains:
+                  ""
+                  <li>zzzz<ol>
+                  ""
+            And I edit the page
+                Then the Raw Editor contains exactly:
+                  ""
+                  <li>zzzz<ol>
+                  ""
+        """
+        content = "<li>zzzz<ol>"
+        self.html_editor.set_content_in_tinymce_editor(content)
+        self.html_editor.save_content()
+        html = self.container_page.content_html
+        self.assertIn(content, html)
+        self.container_page.edit()
+        editor_value = self.html_editor.editor_value
+        self.assertEqual(content, editor_value)
+
+    def test_tinymce_toolbar_buttons_are_as_expected(self):
+        """
+        Scenario: TinyMCE toolbar buttons are as expected
+        Given I have created a Blank HTML Page
+        When I edit the page
+            Then the expected toolbar buttons are displayed
+        """
+        expected_buttons = [
+            u'bold',
+            u'italic',
+            u'underline',
+            u'forecolor',
+            # This is our custom "code style" button, which uses an image instead of a class.
+            u'none',
+            u'alignleft',
+            u'aligncenter',
+            u'alignright',
+            u'alignjustify',
+            u'bullist',
+            u'numlist',
+            u'outdent',
+            u'indent',
+            u'blockquote',
+            u'link',
+            u'unlink',
+            u'image'
+        ]
+        toolbar_dropdowns = self.html_editor.toolbar_dropdown_titles
+        self.assertEqual(len(toolbar_dropdowns), 2)
+        self.assertEqual(['Paragraph', 'Font Family'], toolbar_dropdowns)
+        toolbar_buttons = self.html_editor.toolbar_button_titles
+        self.assertEqual(len(toolbar_buttons), len(expected_buttons))
+
+        for index, button in enumerate(expected_buttons):
+            class_name = toolbar_buttons[index]
+            self.assertEqual("mce-ico mce-i-" + button, class_name)
+
+    def test_static_links_converted(self):
+        """
+        Scenario: Static links are converted when switching between code editor and WYSIWYG views
+        Given I have created a Blank HTML Page
+        When I edit the page
+            And type "<img src="/static/image.jpg">" in the code editor and press OK
+        Then the src link is rewritten to the asset link "image.jpg"
+            And the code editor displays "<p><img src="/static/image.jpg" /></p>"
+        """
+        value = '<img src="/static/image.jpg">'
+        self.html_editor.set_raw_content(value)
+        self.html_editor.save_content()
+        html = self.container_page.content_html
+        self.assertIn('image.jpg', html)
+        self.container_page.edit()
+        self.html_editor.open_raw_editor()
+        editor_value = self.html_editor.editor_value
+        self.assertEqual(value, editor_value)
```

Comment:
the actual test condition that the html tag should convert to a text value is not being tested here. I would recommend using two var string one for html tag and other for text value and then assertEqual both of these where applicable